coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -156,6 +156,7 @@ def ssa_convert(ssa,
             if not os.path.isfile(classes_in):
                 raise ValueError("Path to class labels (%s) does not exist." % \
                     classes_in)
+            if os.path.isfile(classes_in):
```

Comment:
This looks like a bug, thanks for find it out @freedomtan! However, I think there might be a better fix. The whole `if-else` block in https://github.com/apple/coremltools/blob/master/coremltools/converters/nnssa/coreml/ssa_converter.py#L156 need to be rewritten. Also, @bhushan23 do we have any unit test case for `classes_in` argument?

Reply:
No. we don't have any test cases for that